quiz.js

```diff
@@ -94,6 +94,7 @@ function showResults() {
     
   });
 
+  // Start comment section
```

Comment:
The code between the `// Start comment section` and `// End comment section` isn't properly indented.  It's still a part of the `showResults()` function.